### PR TITLE
Update README to say the repo is archived

### PR DESCRIPTION
**Pull request by dipikabh**
I intended to open a PR for the previous change to add this note. Looks like I missed it.

Please take a look to see if everything is okay with this change. Thanks!

Issue tracker: https://github.com/mdn/content-team-projects/issues/55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # canvas-raycaster
-NOTE: This example repository has been archived. The example code and live example can now be found at [mdn/museum/canvas-raycaster](https://github.com/mdn/museum/tree/main/canvas-raycaster) and http://mdn.github.io/museum/canvas-raycaster/.
+> NOTE: This example repository has been archived. The example code and live example can now be found at [mdn/museum/canvas-raycaster](https://github.com/mdn/museum/tree/main/canvas-raycaster) and http://mdn.github.io/museum/canvas-raycaster/.
```
Comment (schalkneethling) [inline README.md]: The link at the end does not work yet. I will need to turn on gh-pages for the museum. Will let you know as soon as that is done.

Comment (schalkneethling) [reply README.md]: We are good to go here now.

Comment (schalkneethling) [inline README.md]:
```suggestion
> NOTE: This example repository has been archived. The example code can now be found at [mdn/museum/canvas-raycaster](https://github.com/mdn/museum/tree/main/canvas-raycaster) and the live example at [Canvas Raycaster Demo](http://mdn.github.io/museum/canvas-raycaster/).
```

**Review by schalkneethling — APPROVED** [review]
One suggested change. Thank you, @dipikabh 🛰️  r+w/c

**Comment by dipikabh** [comment]
Thanks, @schalkneethling for the review 🙏

**Pull request merged**